community/channels.rst

```diff
@@ -54,7 +54,7 @@ Social networks
 - `YouTube <https://www.youtube.com/c/GodotEngineOfficial>`_
 - `Steam <https://steamcommunity.com/app/404790>`_
 
-Forum
+Unofficial Forum
 -----
 
-- `Godot Forums <https://godotforums.org/>`_
+- `Godot unofficial Forums <https://godotforums.org/>`_
```

Comment:
```suggestion
- `Community Forum <https://godotforums.org/>`_
```

As requested in [#777](https://github.com/godotengine/godot-website/pull/777)

Grouping both forums together in a "Forums" category could be better.